frontend/src/components/pages/MultiUserBurndownChart/Presentation.jsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from "react";
+import {
+  LineChart,
+  Line,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  ResponsiveContainer,
+} from "recharts";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+import { makeStyles } from "@material-ui/core/styles";
+import green from "@material-ui/core/colors/green";
+import red from "@material-ui/core/colors/red";
+
+const useStyles = makeStyles((theme) => ({
+  legend: {
+    padding: theme.spacing(1),
+  },
+  selectedFilter: {
+    color: "green",
+    marginRight: "1em"
+  },
+  unselectedFilter: {
+    marginRight: "1em"
+  }
+}));
+
+function Presentation({ userSnapshotArray, authedUser }){
+  const [metricFilter, setMetricFilter] = useState("TotalProjects");
+  const classes = useStyles();
+  return (
+    <React.Fragment>
+      <Typography variant="h6" component="h2">
+        Burndown
+      </Typography>
+      <Typography variant="h6" component="h1">
+        Filter Metric:{" "}
+        <Button
```

Comment:
There will come a time when the metrics change, eg we will want to display total weights, not just total counts.

If we were to introduce one new metric then a whole lot of code would need to change.

How can you make this more DRY?

Something like:

```
const metrics = {
    cardsInCompleteColumnTotalCount: "Complete Cards",
   someOtherDataKey: "User Friendly Title",
}

...

# draw the buttons:

{
    Object.keys(metrics).map(metric => (
           <Button ....>
    ))
}
...
 
# draw the lines:


                  {userSnapshotArray.map((userOnTeam) => (
                    <Line
                      type="monotone"
                      data={userOnTeam.snapshot}
                      dataKey=${selectedDataKey}   ################## why all the if statements? 
                      name={`${userOnTeam.userEmail}`}
                      stroke={userOnTeam.user === authedUser ? green[400] : red[400]}
                      activeDot={{ r: 8 }}
                      strokeWidth={1}
                      dot={false}
                      key={userOnTeam.user}
                    />
                  ))}
                


```